test/test_common/environment.cc

```diff
@@ -328,4 +328,14 @@ void TestEnvironment::setEnvVar(const std::string& name, const std::string& valu
 #endif
 }
 
+void TestEnvironment::unsetEnvVar(const std::string& name) {
+#ifdef WIN32
+  const int rc = ::_putenv_s(name.c_str(), "");
+  ASSERT_EQ(0, rc);
+#else
+  const int rc = ::unsetenv(name.c_str());
+  ASSERT_EQ(rc, 0);
```

Comment:
nit: let's be consistent with others? `ASSERT_EQ(0, rc);`

Reply:
It's  consistent with `TestEnvironment::setEnvVar` implementation. I can fix that one too in the same time?

https://github.com/envoyproxy/envoy/blob/master/test/test_common/environment.cc#L324-L327

Reply:
:+1:

Reply:
Fixed